evaluate.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""torchgeo model evaluation script."""
+
+import argparse
+import csv
+import os
+from typing import Dict, Tuple, Type, Union
+
+import pytorch_lightning as pl
+
+from torchgeo.datasets import (
+    BigEarthNetDataModule,
+    ChesapeakeCVPRDataModule,
+    COWCCountingDataModule,
+    CycloneDataModule,
+    ETCI2021DataModule,
+    LandCoverAIDataModule,
+    NAIPChesapeakeDataModule,
+    RESISC45DataModule,
+    SEN12MSDataModule,
+    So2SatDataModule,
+    UCMercedDataModule,
+)
+from torchgeo.trainers import (
+    BYOLTask,
+    ClassificationTask,
+    MultiLabelClassificationTask,
+    RegressionTask,
+    SemanticSegmentationTask,
+)
+from torchgeo.trainers.chesapeake import ChesapeakeCVPRSegmentationTask
+from torchgeo.trainers.landcoverai import LandCoverAISegmentationTask
+from torchgeo.trainers.naipchesapeake import NAIPChesapeakeSegmentationTask
+from torchgeo.trainers.resisc45 import RESISC45ClassificationTask
+from torchgeo.trainers.so2sat import So2SatClassificationTask
+
+TASK_TO_MODULES_MAPPING: Dict[
```

Comment:
This stuff could just be imported from `train` to reduce code duplication

Reply:
can this all go somewhere in the torchgeo namespace? maybe in __init__.py of trainers?

Reply:
Sure!